From 965e858072279f4b3e3426b57a5cacdd91623826 Mon Sep 17 00:00:00 2001
From: Vishwath Mohan <vishwath@google.com>
Date: Thu, 29 Mar 2018 17:56:33 +0000
Subject: [PATCH 177/306] Revert "Clear pattern on completion (if fade is
 enabled)."

This reverts commit b8e9782f70d6aa28368798cdb969607831c826ce.

Bug: 77257848
Test: Can change from pattern to pin/password

Change-Id: I17f9625e506c8e0839fffa95a62c904890cac603
(cherry picked from commit b03dd4f4a3c5e2a1e2378005ee5e93ca2614dc92)
---
 core/java/com/android/internal/widget/LockPatternView.java | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/core/java/com/android/internal/widget/LockPatternView.java b/core/java/com/android/internal/widget/LockPatternView.java
index 957c784087a..51dd92961f5 100644
--- a/core/java/com/android/internal/widget/LockPatternView.java
+++ b/core/java/com/android/internal/widget/LockPatternView.java
@@ -1008,9 +1008,6 @@ public class LockPatternView extends View {
                 mDrawingProfilingStarted = false;
             }
         }
-        if (mFadePattern) {
-            clearPattern();
-        }
     }
 
     private void cancelLineAnimations() {
-- 
2.17.1

